totalopenstation/output/__init__.py

```diff
@@ -2,6 +2,35 @@
 
 __all__ = ["tops_csv", "tops_dxf", "tops_dat", "tops_sql", "tops_txt", "tops_geojson"]
 
+class Builder:
```

Comment:
I think this is better left out of this PR and postponed to a later release.

Reply:
Yes, ideally a bug report. This is something I want to add in the developer documentation: all changes should follow a procedure of 1. bug report / feature request 2. pull request 3. review 4. merge. Otherwise we are introducing lots of changes in the code without overview and this leads quickly to code that is less maintainable.

Reply:
Created #112 for this purpose and PR #113 .